google.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Google</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="shortcut icon" type="image/x-icon" href="https://www.google.com/favicon.ico">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="google.css">
+    </head>
+    <body>
+        <header>
+            <div class="nav-bar">
+                <a class="nav-item" href="#">About</a>
+                <a class="nav-item" href="#">Store</a>
+            </div>
+
+            <div class="nav-bar">
+                <a class="nav-item" href="#">Gmail</a>
+                <a class="nav-item" href="#">Images</a>
+                <a class="menu nav-item" href="#">
```

Comment:
```suggestion
                <a class="menu nav-item" href="#" aria-label="App menu">
```

This link doesn't have any text in it, and the images inside it are (correctly) aria-hidden.  This means that a screen reader doesn't have a good way to announce this link: voiceover just announces it as "google.html" for me.